test/fakeintake/server/server_test.go

```diff
@@ -264,7 +309,7 @@ func TestServer(t *testing.T) {
 		assert.Equal(t, http.StatusOK, response.StatusCode, "unexpected code")
 	})
 
-	t.Run("should clean payloads older than 15 minutes", func(t *testing.T) {
+	t.Run("should clean payloads older than retention", func(t *testing.T) {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Did you change the default retention?